src/bootstrap/bootstrap.py

```diff
@@ -18,13 +19,26 @@
 
 def get(url, path, verbose=False):
     print("downloading " + url)
-    # see http://serverfault.com/questions/301128/how-to-download
-    if sys.platform == 'win32':
-        run(["PowerShell.exe", "/nologo", "-Command",
-             "(New-Object System.Net.WebClient).DownloadFile('" + url +
-                "', '" + path + "')"], verbose=verbose)
-    else:
-        run(["curl", "-o", path, url], verbose=verbose)
+    for url, path in ((url, path), (url + ".sha256", path + ".sha256")):
+        # see http://serverfault.com/questions/301128/how-to-download
+        if sys.platform == 'win32':
+            run(["PowerShell.exe", "/nologo", "-Command",
+                "(New-Object System.Net.WebClient).DownloadFile('" + url +
+                    "', '" + path + "')"], verbose=verbose)
```

Comment:
While you're here I think this would be much more readable as
```python
"(New-Object System.Net.WebClient).DownloadFile('{}', '{}')".format(url, path)
```

(I think `.format` is more idiomatic than `str + str` anyway)